src/main/java/swiss/sib/rdf/sparql/examples/vocabularies/SIB.java

```diff
@@ -6,6 +6,8 @@
 
 public class SIB {
 	public static final IRI DESCRIBE = SimpleValueFactory.getInstance().createIRI(SHACL.NAMESPACE, "describe");
+	public static final IRI BIGDATA_SELECT = SimpleValueFactory.getInstance().createIRI(SHACL.NAMESPACE, "bigdata_select");
```

Comment:
Why do we need a separate predicate/class for blazegraph select?

Reply:
Because there are many "SPARQL" queries on WikiData that use Blazegraph extensions that are not at all SPARQL 1.1. i.e. we have users outside of SIB now for this code. But that should not have been in this PR. I have accidentally pushed that to this branch :(

Reply:
Should be fixed now with a force push